chaco/tools/simple_zoom.py

```diff
@@ -1,649 +0,0 @@
-""" Defines the SimpleZoom class.
-"""
-
-
-import warnings
-
-warnings.warn(
-    "SimpleZoom has been deprecated, use ZoomTool", DeprecationWarning
-)
-
-from numpy import array
-
-# Enthought library imports
-from enable.api import ColorTrait, KeySpec
-from traits.api import Bool, Enum, Float, Instance, Int, Str, Trait, Tuple
-
-# Chaco imports
-from chaco.abstract_overlay import AbstractOverlay
-from .base_zoom_tool import BaseZoomTool
-from .tool_history_mixin import ToolHistoryMixin
-
-
-class SimpleZoom(AbstractOverlay, ToolHistoryMixin, BaseZoomTool):
```

Comment:
looks like we're using the `SimpleZoom` class in one of the examples - see chaco tutorial, "dynamically controlling interactions" section where `SimpleZoom` is one of the options in the check list editor

Reply:
Ah looks like `ZoomTool` is imported from `chaco.tools.api` but `"SimpleZoom"` is what was listed for the checklist editor.  I've updated the item in the checklist to be `"ZoomTool"`